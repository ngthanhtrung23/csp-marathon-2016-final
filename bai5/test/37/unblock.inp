.ABBCC
.A.DEE
00.DF.
GHHHFI
G.Z.FI
G.ZXXI
