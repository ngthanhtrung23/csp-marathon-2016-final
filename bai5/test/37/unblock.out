43
0 R 1
G U 3
H L 1
I U 1
X R 1
D D 3
0 R 1
A D 1
E L 2
H R 1
G D 1
B L 2
C L 2
F U 2
H R 1
A D 3
0 L 1
E L 1
H L 1
F D 1
C R 1
B R 1
G U 1
H L 1
D U 3
H R 1
G D 3
0 L 1
B L 1
C L 1
E L 1
F U 1
I U 2
H R 2
Z U 3
H L 2
X L 2
F D 3
I D 3
C R 2
D U 1
Z U 1
0 R 4

